Source/Shaders/Materials/PolylineArrowMaterial.glsl

```diff
@@ -13,23 +13,28 @@ float getPointOnLine(vec2 p0, vec2 p1, float x)
 czm_material czm_getMaterial(czm_materialInput materialInput)
 {
     czm_material material = czm_getDefaultMaterial(materialInput);
-    
+
     vec2 st = materialInput.st;
-    
+
+#ifdef GL_OES_standard_derivatives
     float base = 1.0 - abs(fwidth(st.s)) * 10.0;
+#else
+    float base = 1.0;  // FIXME
```

Comment:
It requires screen-space partial derivatives, which we really can't emulate.

Without it, the arrow head will likely be sized in meters, not pixels, which could be an OK fallback.  Try it and see.